prac_06/guitars.py

```diff
@@ -0,0 +1,34 @@
+"""
+Guitars
+Estimated: 35 minutes
+Actual: 28 minutes
+"""
+from prac_06.guitar import Guitar
+
+
+def main():
+    """Add guitar objects to a list and print the attributes in a format."""
+    guitars = []
+    print("My guitars!")
+    name = input("Name: ")
+    while name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: $"))
+        item = Guitar(name, year, cost)
```

Comment:
Maybe this should be called guitar as item might be too generic?

Reply:
Agreed. Using **guitar** also works better for the 'singular in plural' naming convention.